### PR TITLE
Parser created for iosxe show cts

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            }
+        }
+    },
+    "pac_valid_until": "19:56:32 PDT Sep 6 2020",
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py:98]: I understand just one value, but output has section with `====`, so better to have level like below:
```
'pac_summary': {
    pac_info: {
        'pac_valid_until': "19:56:32 PDT Sep 6 2020"
```
device output for this:
```
===================
CTS PAC Summary
=====================
  PAC-Info:
    PAC Valid Until: 19:56:32 PDT Sep 6 2020

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.6": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py:84]: these times, better to have the unit in key name and value is just integer. Please change all where applicable.
```
'idle_time_mins': 60,
'deadtime_secs': 20,
```

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+            "total_sxp_bindings": 3284,
+            "total_active_bindings": 3284
+        },
+        "ipv6": {
+            "total_sxp_bindings": 111,
+            "total_active_bindings": 111
+        }
+    },
+    "cts_role_based_enforcement": "Enabled",
+    "cts_role_based_vlan_enforcement": "Enabled",
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py:261]: these are part of `ip_sgt_bidnings` information. Please change to:
```
"ip_sgt_bindings": {
    (snip)
    "cts_role_based_enforcement": "Enabled",
    "cts_role_based_vlan_enforcement": "Enabled",

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+    "number_trusted_links": 0,
+    "number_untrusted_links": 0
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py:263]: better to have level for respecting section (output with `====`)
```
'trusted_untrusted_links`: {
    "number_trusted_links": 0,
    "number_untrusted_links": 0

**Review by tahigash — CHANGES_REQUESTED** [review]
thanks for your contribution! reviewed schema and left comments.

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py]: Updated

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py (hunk 2 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py]: Updated

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py (hunk 3 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py]: Updated

File: src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py (hunk 4 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py]: Updated

**Review by tahigash — APPROVED** [review]

**Comment by tahigash** [comment]
schema looks good now. @choudnik please review code.

**Review by choudnik — APPROVED** [review]

**Comment by choudnik** [comment]
@qduk, Can you please resolve the conflicts?

**Comment by qduk** [comment]
@choudnik Merge conflicts resolved. Thank you!

**Pull request merged**